db/migrate/20240909042900_create_products.rb

```diff
@@ -1,11 +1,11 @@
-class CreateProducts < ActiveRecord::Migration[7.0]
+class CreateProducts < ActiveRecord::Migration[6.0]
   def change
     create_table :products do |t|
-      t.string :name
-      t.string :description
-      t.decimal :price
+      t.string :name, null: false
+      t.text :description
```

Comment:
Consider adding a validation for the 'name' field in the Product model to ensure data integrity, as it is now marked as 'null: false' in the migration.